Segment-ComScore/Classes/SEGComScoreIntegration.m

```diff
@@ -71,24 +71,30 @@ + (NSDictionary *)mapToStrings:(NSDictionary *)dictionary
 
     [dictionary enumerateKeysAndObjectsUsingBlock:^(NSString *key, NSString *obj, BOOL *stop) {
         id data = [mapped objectForKey:key];
-        if (!![data isKindOfClass:[NSString class]]) {
+        if ([self isDataValid:data]) {
```

Comment:
For type array can call `[array componentsJoinedByString:@"whatever separator"]` if customer doesn't want it in this format: `@"(\n    Dennis,\n    Donald,\n    Bunny\n)"`